src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/ChainPal.cs

```diff
@@ -49,7 +41,7 @@ internal sealed partial class ChainPal : IDisposable, IChainPal
             return status.dwError == 0;
         }
 
-        public IEnumerable<X509ChainElement> ChainElements
+        public X509ChainElement[] ChainElements
```

Comment:
You mean @dotnet-bot, of course :smile:.  The properties on the PAL types for the certificates are memoized by the calling type, and tend to have the same signature as their entrypoint.  So the allocating property is never called twice.

This part of X509 was, shall we say, before my time.  I'd probably have done it as methods, but don't know that changing it is justified.  (I'm merely providing insight here, not trying to add to inertia)